examples/demo_threading/qml/Button.qml

```diff
@@ -19,15 +19,15 @@ Item {
     Image {
         id: inner
         anchors.centerIn: parent
-        source: "./images/activeInner.png"
+        source: "/images/activeInner.png"
```

Comment:
can we use `images/activeInner.png` for these rather than the absolute  `/` ? Or did that not work ?

Reply:
That doesn't work. Not sure why.